docs/installation.md

```diff
@@ -119,15 +144,71 @@ I0222 17:31:14.112689   27922 request.go:668] Waited for 1.046146023s due to cli
 
 Arlon CLI provides an `init` command to install "itself" on a management cluster.
 This command performs a basic setup of `argocd`(if needed) and `arlon` controller.
-If `argocd` is already installed, it assumes that `admin` password is the same as in `argocd-initial-admin-secret` ConfigMap and that `argocd` resides in the `argocd` namespace.
-Similar assumptions are made for detecting Arlon installation as well: assuming that the existence of `arlon` namespace means Arlon controller exists.
-To install Arlon controller using the init command these pre-requisites need to be met:
+It makes the following assumptions while installing `arlon`:
+
```

Comment:
It makes the following assumptions while installing `arlon`:

something missing here?

Reply:
done.